Lib/random.py

```diff
@@ -424,13 +424,14 @@ def sample(self, population, k, *, counts=None):
         # too many calls to _randbelow(), making them slower and
         # causing them to eat more entropy than necessary.
 
-        if isinstance(population, _Set):
-            _warn('Sampling from a set deprecated\n'
-                  'since Python 3.9 and will be removed in a subsequent version.',
-                  DeprecationWarning, 2)
-            population = tuple(population)
         if not isinstance(population, _Sequence):
-            raise TypeError("Population must be a sequence.  For dicts or sets, use sorted(d).")
+            if isinstance(population, _Set):
+                _warn('Sampling from a set deprecated\n'
```

Comment:
Why add `\n` in the middle of a sentence?

Reply:
You’ll have to ask @rhettinger, that’s just a move / reindent of the existing warning.